priv/repo/migrations/20171213153434_add_fields_to_campaigns.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Jingle.Repo.Migrations.AddFieldsToCampaigns do
+  use Ecto.Migration
+
+  def change do
+    rename table(:campaigns), :copy, to: :original_copy
+
+    alter table(:campaigns) do
+      add :edited_copy, :text
+      add :must_say, :text
+      add :due_date, :date
+      add :notes, :string
```

Comment:
Why not `:text`?  Seems like you might end up with > 255 chars of notes.

Reply:
seems reasonable! will change